de_web_plugin.cpp

```diff
@@ -6128,6 +6128,15 @@ void DeRestPluginPrivate::updateSensorNode(const deCONZ::NodeEvent &event)
 
                             if (i->modelId().startsWith(QLatin1String("lumi.vibration")) && i->type() == QLatin1String("ZHAVibration"))
                             {
+
+                                //Set tiltangle to 0 by defaut, but without queueEvent
+                                ResourceItem *item = i->item(RStateTiltAngle);
```

Comment:
Best move this code to the `if (value == 0x0001)` block, so that `state.tiltangle` is set to 0 only on vibration.  It's already set on tilted and dropped, so no need to set it to 0 just before.  Also, here you can issue an event for `state.tiltangle` without getting duplicate events.